packages/document/docs/zh/guide/start/glossary.mdx

```diff
@@ -21,20 +21,21 @@ CSR 是 "Client-Side Rendering"（客户端渲染）的缩写。它表示页面
 - 随着项目迭代应用越来越庞大，难以维护。
 - 跨团队或跨部门协作开发项目导致效率低下的问题。
 
-## Module Federation
+## Modern.js
 
-微前端（Micro-frontend，简称 MFE）是一种类似于微服务的架构，是一种由独立交付的多个前端应用组成整体的架构风格，它将前端应用分解成一些更小、更简单的能够独立开发、测试、部署的应用，而在用户看来仍然是内聚的单个产品。
+Modern.js 是字节跳动 Web 工程体系的开源版本，它提供多个解决方案，来帮助开发者解决不同研发场景下的问题。
 
-它主要解决了两个问题：
+- [Modern.js 代码仓库](https://github.com/web-infra-dev/modern.js)。
 
-- 随着项目迭代应用越来越庞大，难以维护。
-- 跨团队或跨部门协作开发项目导致效率低下的问题。
+## Module Federation
 
-## Modern.js
+模块联邦（Module Federation，简称 MF）是 Webpack 的一个特性。它允许 JavaScript 应用从另一个应用动态加载代码，并在此过程中共享依赖关系。如果使用联邦模块的应用缺少联邦代码所需的依赖项，Webpack 将从该联邦的构建源下载缺失的依赖项。
 
-Modern.js 是字节跳动 Web 工程体系的开源版本，它提供多个解决方案，来帮助开发者解决不同研发场景下的问题。
+这使得可以创建微前端风格的应用程序，多个系统可以共享代码，并在不需要重新构建整个应用程序的情况下进行动态更新。
 
-- [Modern.js 代码仓库](https://github.com/web-infra-dev/modern.js)。
+Modern.js 提供了一个 Module Federation 的示例项目，请参考 [module-federation-examples - modernjs](https://github.com/module-federation/module-federation-examples/tree/db5bdbeee56f779999a2c591fc553eb94eb20b36/modernjs)。
```

Comment:
Rsbuild also provides a MF example, we can use this link: https://github.com/module-federation/module-federation-examples/tree/e82f94a/cra